### DISCUSSION TITLE
Fairwell!

### DISCUSSION THREAD

**Discussion by jsjoeio** [coder/code-server#6031] [General]
It's been a while since I've posted in here but I wanted to share that I'm no longer a maintainer on the project. It's been an amazing 2+ years and I feel grateful to have worked with @code-asher and all the rest of you from the community. I've learned a ton and grown as an engineer in ways I wouldn't have expected. Thanks for all your support and use of the project over the years. I'll be around in the space doing other things and using code-server as a community member.

See you around 👋

**Comment by benz0li** [comment]
@jsjoeio Thank you for all your contributions to this project. Your dedication and open ear for the community will be dearly missed.

All the best for your future endeavours.
